rollup.config.js

```diff
@@ -2,14 +2,28 @@ import path from 'path'
 import babel from '@rollup/plugin-babel'
 import resolve from '@rollup/plugin-node-resolve'
 import typescript from '@rollup/plugin-typescript'
+import esbuild from 'rollup-plugin-esbuild'
 import { sizeSnapshot } from 'rollup-plugin-size-snapshot'
 
 const createBabelConfig = require('./babel.config')
 
 const { root } = path.parse(process.cwd())
-const external = (id) => !id.startsWith('.') && !id.startsWith(root)
+
+const external = (id) =>
+  (!id.startsWith('.') && !id.startsWith(root)) ||
+  id.startsWith('@babel/runtime')
```

Comment:
Though technically a different thing, I'd like to do something more in this build change.

Let's try this, following the valtio setup.
```suggestion
const external = (id) =>
  id.startsWith('./vanilla') ||
  (!id.startsWith('.') && !id.startsWith(root)) ||
  id.startsWith('@babel/runtime')
```